install_osx.sh

```diff
@@ -1,38 +1,19 @@
 #!/bin/sh
```

Comment:
Make a backup of the ~/.zshrc file before writing to it. Also, in order to not overwrite previous backups, name should be unique (probably just include date). 

.zshrc-backup-(day-today)